_haikus/sky-high

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
For the filename, rather than `sky-high` can you name it `sky-high.md` so that it is interpreted by Jekyll properly?